packages/hdwallet-web3-portis/src/web3Portis.ts

```diff
@@ -0,0 +1,214 @@
+import {
+    HDWallet,
+    GetPublicKey,
+    PublicKey,
+    RecoverDevice,
+    ResetDevice,
+    Coin,
+    Ping,
+    Pong,
+    LoadDevice,
+    ETHWallet,
+    ETHGetAddress,
+    ETHSignTx,
+    ETHGetAccountPath,
+    ETHAccountPath,
+    ETHSignMessage,
+    ETHSignedMessage,
+    ETHVerifyMessage,
+    ETHSignedTx
+  } from "@shapeshiftoss/hdwallet-core";
```

Comment:
more bad indentation

Reply:
@keepkeyjon I think we need to add `prettier` as a pre-commit hook 😂

Reply:
😐